includes/modules/shipping/usps.php

```diff
@@ -371,6 +371,10 @@ protected function storefrontInitialization()
             return;
         }
 
+        if ($this->checkCartContents() === false) {
+            return;
+        }
+
```

Comment:
Rather than having a separate 'if' for this condition, please add the additional check to the current outta-here check, after the `$this->checkConfiguration() === false` clause.

Reply:
Done.